opal/mca/common/verbs/common_verbs_port.c

```diff
@@ -68,6 +68,14 @@ int opal_common_verbs_port_bw(struct ibv_port_attr *port_attr,
         /* EDR: 25.78125 Gbps * 64/66, in megabits */
         *bandwidth = 25000;
         break;
+    case 64:
+        /* HDR: 50Gbps * 64/66, in megabits */
+        *bandwidth = 50000;
+        break;
+    case 128:
```

Comment:
1. Yes, either a separate PR or another commit on this PR to fix that segv would be great; thanks.
1. Once you get a patch accepted upstream for the 64/128 values, we can probably take this commit.  I just wouldn't want to take a commit here that is dependent upon a vendor-specific OFED stack (which then might vary between vendors).  Hope that makes sense.
1. I think getting the macros would be an extra bonus, and can likely be a separate upstream patch for libibverbs.  I.e., depending on the timing, either a separate PR or a separate commit on this PR that converts these naked values to enums would be awesome.

Many thanks.